File: apis-test/dotnet/UserService.cs
using System.Collections.Generic;

namespace MyApi.Services
{
    public class UserService
    {
        public object GetUser(string id)
        {
            return new { Id = id, Name = "John Doe", Email = "john@example.com" };
        }

        public IEnumerable<object> ListUsers()
        {
            return new List<object>
            {
                new { Id = 1, Name = "John Doe" },
                new { Id = 2, Name = "Jane Doe" }
            };
        }

        public object UpdateProfile(string userId, dynamic data)
        {
            return new { Id = userId, Data = data, Updated = true };
        }
    }
}

File: apis-test/springboot/UserService.java
package com.example.demo.users;

import org.springframework.stereotype.Service;
import java.util.List;
import java.util.Map;
import java.util.HashMap;

@Service
public class UserService {

    public Map<String, Object> findOne(String id) {
        return Map.of("id", id, "name", "John Doe", "email", "john@example.com");
    }

    public List<Map<String, Object>> findAll() {
        return List.of(
            Map.of("id", 1, "name", "John Doe"),
            Map.of("id", 2, "name", "Jane Doe")
        );
    }

    public Map<String, Object> updateProfile(String userId, Map<String, Object> updateData) {
        Map<String, Object> response = new HashMap<>(updateData);
        response.put("id", userId);
        response.put("updated", true);
        return response;
    }
}
